fedimint-startos/manifest.yaml

```diff
@@ -0,0 +1,96 @@
+id: fedimintd-mutinynet
```

Comment:
To avoid clashing with the future mainnet version and to avoid any potential confusion for users, we're defining the mutinynet package as `fedimintd-mutinynet`. I plan to change to `fedimintd` when we're ready for mainnet.